src/ensembl/src/content/app/species-selector/state/speciesSelectorState.ts

```diff
@@ -60,8 +57,7 @@ const initialState: SpeciesSelectorState = {
     search: LoadingState.NOT_REQUESTED
   },
   ui: {
-    isSelectingStrain: false,
-    isSelectingAssembly: false
+    isSelectingStrain: false
```

Comment:
I think this can be removed as well?

Reply:
There is still some more unused code related to strains in our codebase. I didn't want to dilute the purpose of this PR by removing it as well. I suggest we do it in a separate PR?

Reply:
Sure, happy to leave that to a separate PR.